docker/install-deps

```diff
@@ -91,7 +91,7 @@ if [ -f /usr/bin/apt-get ]; then
         libmecab-dev libncurses5-dev libreadline-dev libpam-dev zlib1g-dev libcurl4-openssl-dev \
         libnuma-dev libjemalloc-dev libc6-dbg valgrind libjson-perl python-mysqldb libevent-dev \
         libmecab2 mecab mecab-ipadic git autoconf libgsasl7 libsasl2-dev libsasl2-modules devscripts \
-        debconf debhelper fakeroot po-debconf psmisc ccache libtool sudo liblz4-dev libedit-dev \
+        debconf debhelper fakeroot po-debconf psmisc ccache libtool sudo liblz4-dev liblz4-tool libedit-dev \
```

Comment:
why tool and not liblz4-dev?

Reply:
> Please show testing, the JIRA issue has the testcase name to run

yes sure, fresh tests results for all OSs: https://ps3.cd.percona.com/view/8.0/job/percona-server-8.0-param/28/console